### PR TITLE
Raw: Commerce Extension

### PR DESCRIPTION
**Pull request by RandyStevens**
### Description
This changeset adds `6/OCS-C`, which defines an extension to the core _OpenCannabis Specification_ to provide structures that express commerce, discounts & currencies.

### Changelog
- [x] Early commerce extension proposal
- [x] All commerce docs and structures moved over
- [x] All commerce structures documented
- [x] Fixes and closes OpenCannabis/RFC#11
- [ ] Review
  - [ ] Retail WG (@OpenCannabis/retail)
  - [x] Technology WG (@OpenCannabis/technology)

### CODE REVIEW

File: 3/3-Core-Structures.md [hunk 1]
```diff
@@ -58,6 +58,30 @@ separately in [Part 1.2, Extension Structures](4-Extension-Structures.md).
...
     - [Website](#opencannabis.contact.Website)
 - `products`: High-level concrete product structures.
+    - [Merchandise](#opencannabis.products.Merchandise)
+    - [MerchandiseFlag](#opencannabis.products.MerchandiseFlag)
+    - [MerchandiseType](#opencannabis.products.MerchandiseType)
+    - [Apothecary](#opencannabis.products.Apothecary)
+    - [ApothecaryType](#opencannabis.products.ApothecaryType)
+    - [Flower](#opencannabis.products.Flower)
+    - [Edible](#opencannabis.products.Edible)
+    - [EdibleIngredient](#opencannabis.products.EdibleIngredient)
```
Comment (tyler-porras) [inline 3/3-Core-Structures.md]: What exactly is edible ingredient? This is the first I am seeing of this in the model.

File: 5/README.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+
+This specification's current status is considered `RAW`, i.e. pre-`DRAFT`. Distribution of this memo is unlimited.
+
+### Abstract
+
+This document describes an extension to the _OpenCannabis Specification, version 1_, that introduces menu-
+related definitions structures, and services that compose, create .
+
+_"Menu"_ in this context, refers to:
+- The contents of a dispensaries products
```
Comment (tyler-porras) [inline 5/README.md]: **dispensaries** should be **dispensary's**

File: 5/README.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+[StaticMenu.FlowersEntry
+  | key:string
+  | value:opencannabis.products.Flower]
+
+
+[StaticMenu.MerchandiseEntry
+  | key:string
+  | value:opencannabis.products.Merchandise]
+
+[StaticMenu.PrerollsEntry
```
Comment (tyler-porras) [inline 5/README.md]: This should be **StaticMenu.PlantsEntry**

File: 5/README.md [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+
+### MenuSettings
+Specifies settings used to generate a menu, or used as input when generating menus.
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| full | [bool](#bool) |  | Flag indicating a full menu, including hidden/out-of-stock items. |
+| keys_only | [bool](#bool) |  | Only include menu keys, no detail data. |
+| snapshot | [opencannabis.crypto.primitives.integrity.Hash](#opencannabis.crypto.primitives.integrity.Hash) |  | Don&#39;t return the menu if it&#39;s identical to this fingerprint. |
+| fingerprint | [opencannabis.crypto.primitives.integrity.Hash](#opencannabis.crypto.primitives.integrity.Hash) |  | Bloom filter to consider when returning or processing menu items. |
```
Comment (tyler-porras) [inline 5/README.md]: :)

**Review by tyler-porras — CHANGES_REQUESTED** [review]
1. Correct the "dispensaries" typo.
2. Correct StaticMenu.PrerollsEntry copy paste error then it looks good.

File: 3/3-Core-Structures.md (hunk 1 above)
Comment (RandyStevens) [reply 3/3-Core-Structures.md]: References ingredients in the edible

File: 5/README.md (hunk 3 above)
Comment (RandyStevens) [reply 5/README.md]: Upon further inspection it looked like i had switched prerolls, and merchandise. switched them back around.

Comment (RandyStevens) [reply 5/README.md]: no i had totally messed it up
good catch

**Comment by RandyStevens** [comment]
Give it a look now @tyler-porras

File: 5/README.md [hunk 5]
```diff
@@ -158,9 +158,9 @@ Specifies a menu, menu data structure, contents, products and so on
   | key:string
   | value:opencannabis.products.Plant]
```
Comment (tyler-porras) [inline 5/README.md]: well, the changes performed now make this wrong. **.products.Plant** should be **.products.Preroll**

**Review by tyler-porras — CHANGES_REQUESTED** [review]

File: 5/README.md (hunk 5 above)
Comment (RandyStevens) [reply 5/README.md]: DANG

**Review by tyler-porras — APPROVED** [review]

File: 5/README.md (hunk 5 above)
Comment (RandyStevens) [reply 5/README.md]: Good catch

**Review by sgammon — APPROVED** [review]
looks good enough for `RAW`

**Pull request merged**